cmd/bootstrap/cmd/pull.go

```diff
@@ -60,14 +59,12 @@ func pull(cmd *cobra.Command, args []string) {
 	log.Info().Msgf("found %d files in google bucket", len(files))
 
 	for _, file := range files {
-		if strings.Contains(file, "node-info.pub") {
-			fullOutpath := filepath.Join(flagOutdir, file)
-			log.Printf("downloading %s", file)
+		fullOutpath := filepath.Join(flagOutdir, file)
```

Comment:
why do we want the script to download the `mainnet-x-execution` folder? That folder is our folder which we create to distribute the root.checkpoint. The check that the script had earlier - if strings.Contains(file, "node-info.pub") {  sounds right to me since it effectively only downloads folders which have that special file node-info.pub